### PR TITLE
Fixes #451 -- add stdlib and runtime deps to the Kotlin project template

### LINKED ISSUE #451: kobaltw --init kotlin should add stdlib and runtime dependencies by default
This is the sort of thing that trips up newbies (aka me as of a couple of weeks ago). No Kotlin program can run without the runtime, and only the very simplest can function without the stdlib, so the Kotlin project template should include them.

### PR DESCRIPTION
**Pull request by MrTact**

### CODE REVIEW

File: src/main/kotlin/com/beust/kobalt/app/LanguageTemplateGenerator.kt [hunk 1]
```diff
@@ -20,6 +20,8 @@ abstract class LanguageTemplateGenerator : ITemplate {
 
     abstract val defaultSourceDirectories : HashSet<String>
     abstract val defaultTestDirectories : HashSet<String>
+    abstract val mainDependencies : ArrayList<Pom.Dependency>
```
Comment (MrTact) [inline src/main/kotlin/com/beust/kobalt/app/LanguageTemplateGenerator.kt]: Could possibly make the case that these should not be abstract & should be initialized with  ArrayList<Pom.Dependency>(), so inheriting generators don't have to do that if they don't need to introduce any dependencies.

File: src/main/kotlin/com/beust/kobalt/app/kotlin/KotlinTemplateGenerator.kt [hunk 2]
```diff
@@ -1,10 +1,15 @@
 package com.beust.kobalt.app.kotlin
 
 import com.beust.kobalt.app.LanguageTemplateGenerator
+import com.beust.kobalt.maven.Pom
 
 class KotlinTemplateGenerator : LanguageTemplateGenerator() {
     override val defaultSourceDirectories = hashSetOf("src/main/kotlin")
     override val defaultTestDirectories = hashSetOf("src/test/kotlin")
+    override val mainDependencies = arrayListOf(
+            Pom.Dependency("org.jetbrains.kotlin","kotlin-runtime", null, "1.1.3"),
```
Comment (cbeust) [inline src/main/kotlin/com/beust/kobalt/app/kotlin/KotlinTemplateGenerator.kt]: Use `Constants.KOTLIN_COMPILER_VERSION` instead of hardcoding.

Comment (MrTact) [inline src/main/kotlin/com/beust/kobalt/app/kotlin/KotlinTemplateGenerator.kt]: Done.

**Comment by cbeust** [comment]
@ethauvin Any other comments before I merge this?

**Comment by ethauvin** [comment]
@cbeust Nope, LGTM.

**Pull request merged**